### ISSUE TITLE
Validator CLI `operator-cli update` Fails

### ISSUE THREAD

**Issue by camzilla1050** [Shardeum/shardeum-bug-reporting#353]
##### What is the issue?

operator-cli update fails update to 1.1.8 version   

```
ode@0d406b6ae5dc:~/app$ operator-cli update
----- STARTING GUI UPDATE PROCESS -----
Your branch is up to date with 'origin/main'.
Already on the latest version.

From https://gitlab.com/shardeum/validator/gui
   67561f0..4312fbd  dev                     -> origin/dev
 + 889fff8...d83f6af feat/confirm_stop_on_active_node -> origin/feat/confirm_stop_on_active_node  (forced update)
   f39f609..fff1ee3  fix/gui_security_issues -> origin/fix/gui_security_issues

----- STARTING CLI UPDATE PROCESS -----
Your branch is up to date with 'origin/main'.
Already on the latest version.

From https://gitlab.com/shardeum/validator/cli
   f6ddf53..b3f7a9d  dev        -> origin/dev

```
```
node@0d406b6ae5dc:~/app$ operator-cli version
runningCliVersion: 0.1.2
minimumCliVersion: 0.1.0
latestCliVersion: 0.1.2
minShardeumVersion: ''
activeShardeumVersion: ''
runningGuiVersion: 0.1.2
minimumGuiVersion: 0.1.0
latestGuiVersion: 0.1.2
runnningValidatorVersion: 1.1.7
```

![Capture d’écran du 2023-04-24 14-13-48](https://user-images.githubusercontent.com/10384623/233993030-1bd47670-47da-4a83-8478-5bbc918f4fd0.png)

**Comment by AntonioZolotoy** [comment]
I've done everything, cli updated, launched, but now the dashboard is empty.
![1](https://user-images.githubusercontent.com/93812394/234065116-49403a89-1cc7-4f79-aa24-0934b0345d95.png)
![2](https://user-images.githubusercontent.com/93812394/234065261-8c54d60a-4024-4ba6-b031-50fb65c6c804.png)
Maintenance, Network, Alert Info, Settings all is empty, no info.

**Comment by camzilla1050** [comment]
@AntonioZolotoy : Maybe just issue with your gui

Now for me is bad to worst..

status:
```
  cause: Error: connect ECONNREFUSED 127.0.0.1:9011
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1146:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '127.0.0.1',
    port: 9011
  }
}
/home/node/app/cli/build/src/utils/fetch-network-data.js:139
    if (nodeInfo === null || nodeInfo.nodeInfo === null) {
                                      ^

TypeError: Cannot read properties of undefined (reading 'nodeInfo')
    at fetchNodeInfo (/home/node/app/cli/build/src/utils/fetch-network-data.js:139:39)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /home/node/app/cli/build/src/node-commands.js:200:34
```
![Capture d’écran du 2023-04-24 20-55-39](https://user-images.githubusercontent.com/10384623/234090440-bf02626d-a838-4f6e-8ced-1ea1273d5671.png)

**Comment by AntonioZolotoy** [comment]
This is all happening because we now have version 1.1.8, but their script immediately installs to version 1.1.8 and that's the problem. even if you delete the node. We need a version of the node 1.1.7 and cli 0.1.3. And where to get version 1.1.7 if you have already updated to 1.1.8 is unclear.

**Comment by Makisblad** [comment]
I have the same problem. is it t still unresolved?

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.
